doc/02-Installation.md

```diff
@@ -0,0 +1,117 @@
+Then use your distribution's package manager to install the `icinga-notifications` package
+or install [from source](02-Installation.md.d/From-Source.md).
+<!-- {% endif %} --><!-- {# end if not icingaDocs #} -->
+
+## Setting up the Database
+
+A MySQL (≥5.5), MariaDB (≥10.1), or PostgreSQL (≥9.6) database is required to run Icinga Notifications.
+Please follow the steps listed for your target database,
+which guide you through setting up the database and user and importing the schema.
+
+### Setting up a MySQL or MariaDB Database
+
+If you use a version of MySQL < 5.7 or MariaDB < 10.2, the following server options must be set:
+
+```
+innodb_file_format=barracuda
+innodb_file_per_table=1
+innodb_large_prefix=1
+```
+
+Set up a MySQL database for Icinga Notifications:
+
+```
+# mysql -u root -p
+
+CREATE DATABASE notifications;
+CREATE USER 'notifications'@'localhost' IDENTIFIED BY 'CHANGEME';
+GRANT ALL ON notifications.* TO 'notifications'@'localhost';
+```
+
+After creating the database, import the Icinga Notifications schema using the following command:
+
+```
+mysql -u root -p notifications < /usr/share/icinga-notifications/schema/mysql/schema.sql
+```
+
+### Setting up a PostgreSQL Database
+
+Set up a PostgreSQL database for Icinga Notifications:
+
+```
+# su -l postgres
+
+createuser -P notifications
+createdb -E UTF8 --locale en_US.UTF-8 -T template0 -O notifications notifications
+echo 'CREATE EXTENSION IF NOT EXISTS citext;' | psql notifications
+```
+
+The `CREATE EXTENSION` command requires the `postgresql-contrib` package.
+
+Edit `pg_hba.conf`, insert the following before everything else:
+
+```
+local all notifications           md5
+host  all notifications 0.0.0.0/0 md5
+host  all notifications      ::/0 md5
+```
+
+To apply these changes, run `systemctl reload postgresql`.
+
+After creating the database, import the Icinga Notifications schema using the following command:
+
+```
+psql -U icinga-notifications notifications < /usr/share/icinga-notifications/schema/pgsql/schema.sql
+```
+
+## Configuring Icinga Notifications
+
+Icinga Notifications installs its configuration file to `/etc/icinga-notifications/config.yml`,
+pre-populating most of the settings for a local setup. Before running Icinga Notifications,
+adjust the database credentials and the Icinga Web 2 URL.
+The configuration file explains general settings.
+All available settings can be found under [Configuration](03-Configuration.md).
+
+## Configuring Icinga 2
+
+There is no special configuration required for Icinga 2 on its own.
+However, you may want to use an Icinga 2 instance as a notification source.
+This requires configuring an `ApiUser` as described later in the
+[Icinga Notifications Web documentation](https://icinga.com/docs/icinga-notifications-web/latest/doc/03-Configuration/).
+At this point please continue with this manual until the Icinga Notification Web documentation leads there.
```

Comment:
Do we need this section at all? This sounds more confusing than helpful to me. It has a reference to Icinga Notifications Web, but doesn't want you to follow it but instead ignore it and continue with the following step here anyways.

Reply:
I thought that keeping this section with a reference to the web part was the consensus of the discussion earlier. But yes, it also felt weird for me, that's why I added the last sentence, stating that this doesn't really matter at this point.

However, I think I am going to remove this here.